bdd-generator/src/main/java/com/epam/test_generator/config/WebConfig.java

```diff
@@ -17,21 +12,19 @@
 
 /**
  * Configuration class customizing the default Java-based configuration for Spring MVC.
+ *
  * @see WebMvcConfigurer
  */
 @Configuration
-@EnableWebMvc
 @ComponentScan("com.epam.test_generator")
-@PropertySource(value = "classpath:email.properties")
+@EnableWebMvc
 public class WebConfig implements WebMvcConfigurer {
 
-    @Resource
-    private Environment environment;
-
     /**
-     * Specialization of PlaceholderConfigurerSupport that resolves ${...} placeholders
-     * within bean definition property values and @Value annotations against the
-     * current Spring Environment and its set of PropertySources.
+     * Specialization of PlaceholderConfigurerSupport that resolves ${...} placeholders within bean
+     * definition property values and @Value annotations against the current Spring Environment and
+     * its set of PropertySources.
+     *
      * @return property sources placeholder configurer bean
      */
     @Bean
```

Comment:
I doubt if this bean still required, please investigate and answer here why it is needed/remove it